package/MDAnalysis/analysis/encore/dimensionality_reduction/src/spe.c

```diff
@@ -22,10 +22,12 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-//#include <math.h>
+#include <math.h>
 #include <time.h>
 #include <sys/types.h>
 #include <time.h>
+#include <sys/types.h>
+#include <unistd.h>
```

Comment:
sorry for the double include - I'll remove one

`unistd.h` is necessary for getpid() (line 190)

the inclusion of math.h fixes another warning:

`MDAnalysis/analysis/encore/dimensionality_reduction/src/spe.c:73:12: warning: implicit declaration of function ‘sqrt’ [-Wimplicit-function-declaration]`